src/WrappedMToken.sol

```diff
@@ -247,13 +258,15 @@ contract WrappedMToken is IWrappedMToken, Migratable, ERC20Extended {
     }
 
     /// @inheritdoc IWrappedMToken
-    function claimOverrideRecipientFor(address account_) public view returns (address recipient_) {
+    function claimRecipientFor(address account_) public view returns (address recipient_) {
         return
-            address(
-                uint160(
-                    uint256(_getFromRegistrar(keccak256(abi.encode(CLAIM_OVERRIDE_RECIPIENT_KEY_PREFIX, account_))))
-                )
-            );
+            (_accounts[account_].hasClaimRecipient)
```

Comment:
consider adding this logic here
```
address claimOverrideRecipient_ = claimRecipientFor(account_);
address claimRecipient_ = claimOverrideRecipient_ == address(0) ? account_ : claimOverrideRecipient_;```
```

Reply:
Also, let's confirm the order once again whether the governance has first priority, then owner of account, then default.

Reply:
Governance does not have first priority because:
- it would allow governance to retroactively steal unclaimed yield
- it costs more gas